.github/workflows/build-coredns-images.yaml

```diff
@@ -0,0 +1,68 @@
+name: Build and Publish CoreDNS Images
+
+on:
+  push:
+    branches:
+      - main
+      - coredns_plugin
```

Comment:
@mikenairn Remove this before merge

Reply:
Also double check the image is working correctly before merging

Reply:
Was an issue with the image, Dockerfile had to be updated to include an additional stage to allow the coredns binary to work correctly in nonroot containers https://github.com/Kuadrant/dns-operator/blob/867c019829f5fe09d772ef82be6f4dcf0161a576/coredns/plugin/Dockerfile.

Based on the the [CoreDNS Dockerfile](https://github.com/coredns/coredns/blob/master/Dockerfile).